models/DataObject/ClassDefinition/Data/Wysiwyg.php

```diff
@@ -114,7 +114,7 @@ public function getDataForResource(mixed $data, DataObject\Concrete $object = nu
     public function getDataFromResource(mixed $data, DataObject\Concrete $object = null, array $params = []): ?string
     {
         if (is_string($data)) {
-            $data = self::getWysiwygSanitizer()->sanitize($data);
+            $data = html_entity_decode(self::getWysiwygSanitizer()->sanitize($data));
```

Comment:
```suggestion
            $data = self::getWysiwygSanitizer()->sanitize($data);
```
as far as i see, this change is unneccesary, leaving the suggestion here

Reply:
@kingjia90 Did you try to get the value in API? I was able to reproduce when calling the getter on API

Reply:
ah, got it now, you're right, it's broken in eg. `object_localized_query_CAR_en` , i was checking `object_localized_data_CAR`